images/nginx-slim/build.sh

```diff
@@ -207,8 +207,8 @@ WITH_MODULES="--add-module=$BUILD_PATH/ngx_devel_kit-$NDK_VERSION \
   --add-module=$BUILD_PATH/nginx-goodies-nginx-sticky-module-ng-$STICKY_SESSIONS_VERSION \
   --add-module=$BUILD_PATH/nginx-http-auth-digest-$NGINX_DIGEST_AUTH \
   --add-module=$BUILD_PATH/ngx_http_substitutions_filter_module-$NGINX_SUBSTITUTIONS \
-  --add-module=$BUILD_PATH/nginx-opentracing-$NGINX_OPENTRACING/opentracing \
-  --add-module=$BUILD_PATH/nginx-opentracing-$NGINX_OPENTRACING/zipkin"
+  --add-dynamic-module=$BUILD_PATH/nginx-opentracing-$NGINX_OPENTRACING_VERSION/opentracing
+  --add-dynamic-module=$BUILD_PATH/nginx-opentracing-$NGINX_OPENTRACING_VERSION/zipkin"
```

Comment:
Please check https://github.com/rnburn/ingress-nginx/blob/c26af267c5a3d2546ace345122b0fc007a287204/images/nginx-slim/build.sh#L243
you need to copy the opentracing and zipkin modules

Reply:
`make install` does copy the modules by default into whatever `--modules-path` is set to or `/usr/share/nginx/modules` by default. 

If you want the modules installed to `/etc/nginx/modules/`, any reason you don't just set `--with-modules-path=/etc/nginx/modules`? If you look at the image, you can see you have `ngx_http_modsecurity_module.so` installed twice:
```
root@cb1282564228:/# ls /usr/share/nginx/modules/
ngx_http_modsecurity_module.so	ngx_http_opentracing_module.so	ngx_http_zipkin_module.so
root@cb1282564228:/# ls /etc/nginx/modules/
ngx_http_modsecurity_module.so
```

Reply:
@rnburn you are right. Please change --with-modules-path=/etc/nginx/modules and remove the cp command